.github/workflows/syntax_check_cloud.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
+        os: [ubuntu-22.04, ubuntu-24.04]
```

Comment:
I guess we could also update this to only run on ubuntu-latest (latest LTS), it doesn't really make sense to do syntax checking on multiple Ubuntu versions. What do you guys think?

Reply:
fixed! @tmoerkerken could you have a look at this PR, so we don't see the ubuntu 20 deprecation warning anymore on the other open PRs